pkg/operator/assets/bindata.go

```diff
@@ -329,26 +329,19 @@ spec:
         terminationMessagePolicy: FallbackToLogsOnError
         image: ${IMAGE}
         imagePullPolicy: IfNotPresent
-        command: ["/bin/bash", "-ec"]
+        command: ["oauth-apiserver", "start"]
         args:
-          - |
-            if [ -s /var/run/configmaps/trusted-ca-bundle/tls-ca-bundle.pem ]; then
```

Comment:
it was taken from `oas` - https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/bindata/v3.11.0/openshift-apiserver/deploy.yaml#L67

should we update there as well?

Reply:
no, I think OAS is actually using this, but we're not